index.js

```diff
@@ -1,11 +1,5 @@
 'use strict';
+const getFromEnv = parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;
+const isEnvSet = 'ELECTRON_IS_DEV' in process.env;
 
-const getFromEnv = () => {
-	if ('ELECTRON_IS_DEV' in process.env) {
-		return parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;
-	}
-
-	return false;
-};
-
-module.exports = getFromEnv() || process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath);
+module.exports = isEnvSet ? getFromEnv : process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath);
```

Comment:
@kevva if you mind, according to me wrapping the `!isEnvSet` part of the ternary around `()` could improve readability....